packages/core/src/env-set/index.ts

```diff
@@ -17,6 +17,7 @@ export enum UserApps {
   Api = 'api',
   Oidc = 'oidc',
   DemoApp = 'demo-app',
+  AccountCenter = 'account-center',
```

Comment:
Need to set proper CSP headers in the `koaSecurityHeaders` middleware for this newly added app.

Reply:
updated